ui/css/logviewer.css

```diff
@@ -143,20 +147,11 @@ body {
     font-size: 11px;
 }
 
-.logviewer-step.selected {
-    color: #232323;
+.lv-step.selected {
     background-color: #FFF;
 }
 
-.logviewer-step.btn-success.selected {
-    border-color: #398439;
-}
-
-.logviewer-step.btn-warning.selected {
-    border-color: #d58512;
-}
```

Comment:
I checked and re-checked, and I think these this .double.classes with `btn-warning` are unused.